### ISSUE TITLE
X_TranslateCoords error on 3.3.1 (and master)

### ISSUE THREAD

**Issue by larsoner** [glfw/glfw#1620]
Over at VisPy we are using GLFW and noticed [some X failures](https://travis-ci.org/vispy/vispy/jobs/633974644#L7788) on Linux x86_64 platforms (at least). When using a window without decoration (with decoration things are fine), I get an X Error code:
```
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  40 (X_TranslateCoords)
  Resource id in failed request:  0x0
  Serial number of failed request:  242
  Current serial number in output stream:  242
```
`git bisect` tells me that the offending commit is:

https://github.com/glfw/glfw/commit/fe57e3c2921a1901390534e1e51053df70b5644b

Which at least makes some sense since there is a call to XTranslateCoordinates in there.

I am a X and GLFW novice and my C debugging skills are not the best, but happy to help provide more information or try anything that might help.

**Comment by Luflosi** [comment]
This was also reported in https://github.com/kovidgoyal/kitty/issues/2255 after I ported this commit to kitty.

**Comment by elmindreda** [comment]
This is bad enough to warrant 3.3.2 on its own.

**Comment by elmindreda** [comment]
This is hopefully fixed now with 9372ba95faabd1bb45280d201d6ccdae77bf872e.

**Comment by djhoese** [comment]
Wow, this is great! Since you mentioned a 3.3.2 release, are there plans for a release soon? If not, the vispy group may be able to include a patch on the glfw conda-forge package as a workaround.

**Comment by elmindreda** [comment]
My intention is to put it out this week, possibly including a couple of other fixes.

**Comment by larsoner** [comment]
I can confirm that the bug is fixed in the 3.3-stable branch, thanks for the quick fix!
